fastlane/lanes/build.rb

```diff
@@ -464,4 +495,41 @@ def upload_build_to_app_center(
       notify_testers: false
     )
   end
+
+  def upload_gutenberg_sourcemaps(sentry_project_slug:, release_version:, build_version:, app_identifier:)
+    # The bundle and source map files are the same for all architectures.
+    gutenberg_bundle = File.join(PROJECT_ROOT_FOLDER, "Pods/Gutenberg/Frameworks/Gutenberg.xcframework/ios-arm64/Gutenberg.framework")
+
+    Dir.mktmpdir do |sourcemaps_folder|
+      # It's important that the bundle and source map files have specific names, otherwise, Sentry 
+      # won't symbolicate the stack traces.
+      FileUtils.cp(File.join(gutenberg_bundle, 'App.js'), File.join(sourcemaps_folder, 'main.jsbundle'))
+      FileUtils.cp(File.join(gutenberg_bundle, 'App.composed.js.map'), File.join(sourcemaps_folder, 'main.jsbundle.map'))
+
+      # To generate the full release version string to attach the source maps, we need to specify:
+      # - App identifier
+      # - Release version
+      # - Build version
+      # This conforms to the following format: <app_identifier>@<release_version>+<build_version>
+      # Here are a couple of examples:
+      # - Prototype build: com.jetpack.alpha@24.2+pr22654-07765b3 
+      # - App Store build: org.wordpress@24.1+24.1.0.3
+
+      sentry_upload_sourcemap(
+        auth_token: get_required_env('SENTRY_AUTH_TOKEN'),
+        org_slug: SENTRY_ORG_SLUG,
+        project_slug: sentry_project_slug,
+        version: release_version,
+        dist: build_version,
+        build: build_version,
+        app_identifier: app_identifier,
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/wordpress-mobile/WordPress-iOS/blob/a1a692f09379dc1e081781d78ae488238eb956a6/fastlane/lanes/build.rb#L525"></span>Style/HashSyntax: Omit the hash value.</td>
    </tr>
  </tbody>
</table>